frame/staking/src/pallet/mod.rs

```diff
@@ -477,10 +477,6 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
-	/// The earliest era for which we have a pending, unapplied slash.
-	#[pallet::storage]
-	pub(crate) type EarliestUnappliedSlash<T> = StorageValue<_, EraIndex>;
```

Comment:
don't we need some storage migration in case there is some value stored while updating runtime?

Reply:
sorry, I'm just hypersensitive about storage changing :see_no_evil:

Reply:
https://github.com/paritytech/substrate/pull/11823/commits/c831ff8f2887110f38747265e9e21031f6ed2b63